### PR TITLE
21 sunsaver logger

### PR DESCRIPTION
**Pull request by EliasMartinezJ**
closes issue #21

### CODE REVIEW

File: dencam/sunsaver_log.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        volt_ld = str(math.trunc(SunSaver.read_register(10) * 100 * 2**-(15)))
+        curr_chrg = str(math.trunc(SunSaver.read_register(11) * 79.16 * 2**-(15)))
+        curr_ld = str(math.trunc(SunSaver.read_register(12) * 79.16 * 2**-(15)))
+        temp_amb = str(SunSaver.read_register(15))
+        temp_rts = str(SunSaver.read_register(16))
+        charge_state = str(SunSaver.read_register(17))
+        ah_charge = str(SunSaver.read_register(45) * 0.1)
+        ah_load = str(SunSaver.read_register(46) * 0.1)
+        alarm = SunSaver.read_register(50)
+        alarm_list = ["RTS open", "RTS shorted", "RTS disconnected",
```
Comment (tkswanson) [inline dencam/sunsaver_log.py:42]: variables like these lists can be declared outside of the function as global variables so the pi doesnt have to create them every time the function is called

**Review by tkswanson — CHANGES_REQUESTED** [review]
Just out of curiosity, is there are reason for having two scripts? mppt.py and sunsaver_log.py
Also, it doesn't look like this includes the font size change yet, correct?

**Comment by EliasMartinezJ** [comment]
Was actually gonna ask, would I be able to merge these two scripts? And
just call the sole method that gets text for solar display without worrying
about the logger running? Since I call it at the bottom of the script.

Also right,  Im just about to push this new commit that fixes that .

I'll attach an image of how it looks now

**Review by icr-ctl — APPROVED** [review]
Rock on!

**Pull request merged**